src/SerializeAST.h

```diff
@@ -0,0 +1,19 @@
+//
+//  SerializeAST.h
+//  snowcrash
+//
+//  Created by Pavan Kumar Sunkara on 18/01/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+
+#ifndef SNOWCRASH_SERIALIZE_AST_H
+#define SNOWCRASH_SERIALIZE_AST_H
+
+#include "Serialize.h"
+
+namespace snowcrash {
+
+    sos::Object wrapBlueprint(const Blueprint& blueprint);
```

Comment:
So you are introducing a new convention here. Up until now we have had `loverCamelCase` only methods of classes / structures. We are using `UpperCamelCase` for standalone functions. 

I know you are chasing the coding standards so please align this.